sdk/nextjs/src/server/ably.ts

```diff
@@ -0,0 +1,32 @@
+import { unstable_cache } from 'next/cache'
+
+const sha1 = async (str: string, subtle: SubtleCrypto) => {
+    const source = new TextEncoder().encode(str)
+    const result = await subtle.digest(
+        {
+            name: 'SHA-1',
+        },
+        source,
+    )
+
+    return Array.from(new Uint8Array(result))
+        .map((b) => b.toString(16).padStart(2, '0'))
+        .join('')
+}
+
+const getChannelName = async (key: string, subtle: SubtleCrypto) => {
+    return `dvc_client_${await sha1(`${key}`, subtle)}_v1`
+}
+
+export const sseURlGetter = (
```

Comment:
are we okay with manually constructing the ably URL like this in a hard-coded way on the SDK? The alternatives are to:
- put a CF worker in front of our CDN config and dynamically add the URL there
- add the URL to the static output of our CDN config, rather than dynamically adding it to the bucketed config